website/app/components/Head.tsx

```diff
@@ -0,0 +1,39 @@
+import { Helmet } from 'react-helmet';
+import { DocumentationLoader } from '~/loaders/documentation.server';
+import codeHikeStyles from '@code-hike/mdx/dist/index.css';
+
+export const Head = ({ data }: { data: DocumentationLoader }) => {
+  const favicon = getFavicon({ data });
+
+  return (
+    //@ts-ignore
```

Comment:
Why ts-ignore?

Reply:
react-helmet was giving a type error, it wasn't a JSX element apparently, not sure how to fix this